README.md

```diff
@@ -44,6 +44,24 @@ Supervisor.start_link(
 )
 ```
 
+In the Phoenix web framework, you can find this in `lib/<myapp>/<myapp>.ex`.
```

Comment:
I wouldn't say this example is in fact Phoenix specific. The default path of the app file is the same for non-Phoenix OTP apps (since there are no Phoenix apps, there are only OTP apps). 

I also think instead of the whole example (which is very similar to the one above), we could just have some form of this sentence which points out people to the typical location of the top-level supervisor. Something like:

> If you want to do this in a top-level supervisor, it can usually be found at `lib/<myapp>.ex`.

And then skip the mostly duplicated example. Would that be fine?